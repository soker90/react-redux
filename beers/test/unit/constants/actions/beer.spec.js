// beer.spec.js
import beersConstants from '../../../../src/constants/actions/beer'

describe('Beer constants', function () {
  test('exist', () => {
    expect(beersConstants).toMatchSnapshot()
  })
})
