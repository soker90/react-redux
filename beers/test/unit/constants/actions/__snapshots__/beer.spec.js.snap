exports[`Beer constants exist 1`] = `
Object {
  "ADD_BEER": "ADD_BEER",
  "ADD_BEER_SUCCESS": "ADD_BEER_SUCCESS",
  "BEER_ERROR": "BEER_ERROR",
  "FETCH_BEER": "FETCH_BEER",
  "FETCH_BEER_SUCCESS": "FETCH_BEER_SUCCESS",
  "REMOVE_BEER": "REMOVE_BEER",
  "REMOVE_BEER_SUCCESS": "REMOVE_BEER_SUCCESS",
}
`;
